SofaKernel/modules/SofaHelper/src/sofa/helper/GenerateRigid.h

```diff
@@ -178,7 +178,7 @@ void generateRigid(Rigid3MassType& mass, type::Vector3& center, io::Mesh *mesh
 
     if( rotation != type::Vector3(0,0,0) ) {
 
-        type::Quaternion q = sofa::helper::Quater<SReal>::createQuaterFromEuler( rotation*M_PI/180.0 );
+        type::Quaternion q = type::Quaternion::createQuaterFromEuler( rotation*M_PI/180.0 );
```

Comment:
Didn't we said that we should stop using Quatenrion and use Quat, Quatf, Quatd instead ?